OOP/OOP6_ResizableArray/src/test/java/LukCharListTest.java

```diff
@@ -0,0 +1,62 @@
+import static org.junit.Assert.*;
+
+public class LukCharListTest {
+
+    @org.junit.Test
+    public void save() {
+        //given
+        final LukCharList lukCharList = new LukCharList();
+        final int expectedIntPosition = 0;
+        //when
+        final int actualIntPosition = lukCharList.save('5');
+        //then
+        assertEquals(actualIntPosition,expectedIntPosition);
+    }
+
+    @org.junit.Test
+    public void should_check_index_value() {
+        //given
+        final LukCharList lukCharList = new LukCharList();
+        lukCharList.save('5');
+        final char expectedChar = '5';
+        //when
+        final char actualChar = lukCharList.getByIndex(0);
+        //then
+        assertEquals(actualChar, expectedChar);
+    }
+
+    @org.junit.Test
+    public void should_check_all_array() {
+        //given
+        final LukCharList lukCharList = new LukCharList();
+        lukCharList.save('5');
+        final char[] expectedCharArray = {'5'};
+        //when
+        final char[] actualCharArray = lukCharList.getAll();
+        //then
+        assertArrayEquals(expectedCharArray, actualCharArray);
+    }
```

Comment:
This test is ok, but I would write another one or include more assertions for larger number of records